spec/components/schemas.yaml

```diff
@@ -818,6 +818,150 @@ components:
       description: Feature identifier (unique per collection)
       example: smr200-200-4-2019
       type: string
+    itemGeometry:
+      oneOf:
+        - title: GeoJSON Point
+          type: object
+          required:
+            - type
+            - coordinates
+          properties:
+            type:
+              type: string
+              enum:
+                - Point
+            coordinates:
+              type: array
+              minItems: 2
+              items:
+                type: number
+            bbox:
+              type: array
+              minItems: 4
+              items:
+                type: number
+        - title: GeoJSON LineString
+          type: object
+          required:
+            - type
+            - coordinates
+          properties:
+            type:
+              type: string
+              enum:
+                - LineString
+            coordinates:
+              type: array
+              minItems: 2
+              items:
+                type: array
+                minItems: 2
+                items:
+                  type: number
+            bbox:
+              type: array
+              minItems: 4
+              items:
+                type: number
+        - title: GeoJSON Polygon
+          type: object
+          required:
+            - type
+            - coordinates
+          properties:
+            type:
+              type: string
+              enum:
+                - Polygon
+            coordinates:
+              type: array
+              items:
+                type: array
+              minItems: 4
+              items:
+                type: array
+                minItems: 2
+                items:
+                  type: number
+            bbox:
+              type: array
+              minItems: 4
+              items:
+                type: number
```

Comment:
There is already a schema for the `bbox` that should be reused here and in any GeoJSON types. Its description is not 100% correct as it is currently used in the item response schemas and in item query parameter. For the response the description sentence `Only features that have a geometry that intersects the bounding box are selected.` doesn't make sense and should be removed. However this sentence make sense for the query parameter. But for this one we can use the `bboxfilter` shemas instead.

Reply:
why not